package.json

```diff
@@ -28,5 +28,12 @@
   },
   "pre-commit": [
     "test"
-  ]
+  ],
+  "dependencies": {
+    "babel-core": "^6.4.0",
+    "babel-preset-es2015": "^6.3.13",
```

Comment:
Maybe use [`babel-preset-env`](https://github.com/babel/babel-preset-env) instead? It's future-proof.